tests/ha/test_data_pod_failure.py

```diff
@@ -1371,174 +1321,167 @@ def test_reads_deletes_during_pod_down(self):
+            new_s3data[bkt] = s3_data_del[bkt]
 
         args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
                 'skipput': True, 'skipdel': True, 's3_data': new_s3data, 'di_check': True,
                 'output': rd_output}
         self.ha_obj.put_get_delete(event, s3_test_obj, **args)
-        rd_resp = ()
+        rd_resp = tuple()
         while len(rd_resp) != 4:
             rd_resp = rd_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
         if not rd_resp:
-            assert_utils.assert_true(False, "Failed to READ/Verify remaining buckets.")
+            assert_utils.assert_true(False,
+                                     "Failed to get READ/Verify response on remaining buckets.")
         event_bkt_get = rd_resp[0]
         fail_bkt_get = rd_resp[1]
         event_di_bkt = rd_resp[2]
         fail_di_bkt = rd_resp[3]
 
         # Above four lists are expected to be empty as all pass expected
         assert_utils.assert_false(len(fail_bkt_get) or len(fail_di_bkt) or len(event_bkt_get) or
-                                  len(event_di_bkt), "Expected pass in read and di check "
-                                                     "operations. Found failures in READ: "
-                                                     f"{fail_bkt_get} {event_bkt_get}"
-                                                     f"or DI_CHECK: {fail_di_bkt} {event_di_bkt}")
-        LOGGER.info("Step 6: Successfully verified READs and DI check for remaining buckets: %s",
-                    buckets)
+                                  len(event_di_bkt),
+                                  "Expected pass in READs & DI_CHECK operations. Found failures in"
+                                  f"READ: {fail_bkt_get} {event_bkt_get} OR in "
+                                  f"DI_CHECK: {fail_di_bkt} {event_di_bkt}")
+        LOGGER.info("Step 4: Successfully checked READs and DI_CHECK for remaining buckets.")
 
-        LOGGER.info("Step 7: Deleting remaining buckets.")
+        LOGGER.info("Step 5: Deleting remaining %s buckets.", len(remain_bkts))
         args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
                 'skipput': True, 'skipget': True, 'output': del_output}
         self.ha_obj.put_get_delete(event, s3_test_obj, **args)
-        del_resp = ()
+        del_resp = tuple()
         while len(del_resp) != 2:
             del_resp = del_output.get(timeout=HA_CFG["common_params"]["60sec_delay"])
         if not del_resp:
             assert_utils.assert_true(False, "Failed to DELETE remaining buckets.")
         event_del_bkt = del_resp[0]
         fail_del_bkt = del_resp[1]
         assert_utils.assert_false(len(event_del_bkt) or len(fail_del_bkt),
-                                  f"Failed to delete buckets: {event_del_bkt} and {fail_del_bkt}")
+                                  f"Failed to DELETE buckets: {event_del_bkt} OR {fail_del_bkt}")
```

Comment:
why OR?
and was fitting correct in the statement.

Reply:
> why OR? and was fitting correct in the statement.

Its OR only in assert condition check.

Reply:
This is logger not assert condition.
We have to log like, Failed buckets are ... and ... .